src/components/common/ProductBox/ProductBox.module.scss

```diff
@@ -16,6 +16,16 @@
     }
     overflow: hidden;
     align-items: flex-end;
+
+    .image {
+      position: absolute;
```

Comment:
absolute nie jest potrzebne, zamiast tego popraw istniejący style innych elementów aby zdjęcie wypełniło całość dostępnego miejsca

Reply:
ale czemu nie uzywac absolute? nie za bardzo rozumiem...

Reply:
a po co? absolute jest przydatny kiedy potrzebujemy wyświetlić coś nad innym elementem, nad rodzeństwem albo nad rodzicem, niestandardowo. Tutaj nie ma takiej sytuacji. 
Natomiast kiedy stosujesz absolute to musz dbać o to aby zawsze wyświetlał się poprawnie, kod jest też mniej przewidywalny, bo widzisz na stronie standardowy element i nie spodziewasz się że tam jest jakiś padding a nad tym paddingiem znajduje się zdjęcie ustawione za pomocą absolute.